src/app/content/highlights/components/cardUtils.ts

```diff
@@ -9,13 +9,26 @@ import { assertDefined } from '../../../utils/assertions';
 import { cardMarginBottom } from '../constants';
 import { HighlightData } from '../types';
 
+export const hiddenHighlightOffset = -9999;
+
 export const getHighlightOffset = (container: HTMLElement | undefined, highlight: Highlight) => {
   if (!container || !highlight.range || !highlight.range.getBoundingClientRect) {
     return;
   }
 
   const {top, bottom } = highlight.range.getBoundingClientRect();
 
+  const highlightElement = highlight.elements[0] as HTMLElement;
+  const expandableAncestor = highlightElement
+    && highlightElement.closest('[data-type="solution"][aria-expanded="false"]');
+
+  if (expandableAncestor) {
+    return {
+      bottom: hiddenHighlightOffset,
```

Comment:
Instead of calculating the offset including parent/scroll offesets, it just returns -9999, which will hide the highlight offscreen. 
Before this change, it would return the offsets of the highlight normally. Even tho, that highlight might have been hidden by collapsed solution wrapper.